helm/docs-app/templates/pdb.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Capabilities.APIVersions.Has "policy/v1/PodDisruptionBudget" }}
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ include "resource.dex.name" . }}
```

Comment:
It seems like this isn't working - helm fails to find `resource.dex.name`

Reply:
Yeah, that was a dull mistake